cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +200,94 @@
   # bytes for the OCSP response.
   #AllowTestCert = false
 
+# IMPORTANT NOTE: the support of the ACME protocol and automatic renewal of
+# certificates is currently in the EXPERIMENTAL stage.  Once we have more
+# experience with people using it out in the wild, we will gradually move it to
+# the PRODUCTION stage.
+#
+# Configure ACME, which allows the webpkgserver to automatically renew Signed
+# Exchange certificates.
+#
+# ACME is a protocol that allows automatic renewal of certificates.
+# webpkgserver uses the ACME library from github.com/go-acme/lego
```

Comment:
"webpkgserver uses the ACME library ..." is probably not very important for those who just use webpkgserver, especially those who don't set up DNSProvider.

"ACME is a protocol that allows automatic certificate renewal. Setting the Enable parameter to true will make webpkgserver automatically request a renewed certificate whenever it has determined that the current certificate is expired or about to expire.

If you set up more than one instance of webpkgserver, ...

webpkgserver uses the ACME library from github.com/go-acme/lego.

For the full ACME spec, see https://tools.ietf.org/html/rfc8555."

Reply:
Done.